nextflow_schema.json

```diff
@@ -119,7 +119,6 @@
                 },
                 "observations_name_col": {
                     "type": "string",
-                    "default": "sample",
```

Comment:
Maybe mention in the description that we will now default to the same column as the ID?

Reply:
done!